apps/avifenc.c

```diff
@@ -818,6 +805,62 @@ int main(int argc, char * argv[])
         ++argIndex;
     }
 
+    // Check lossy/lossless parameters and set to default if needed.
+    if (lossless) {
+        // Pixel format.
+        if (input.requestedFormat != AVIF_PIXEL_FORMAT_NONE && input.requestedFormat != AVIF_PIXEL_FORMAT_YUV444) {
+            fprintf(stderr,
+                    "When set, the pixel format can only be 444 in lossless "
+                    "mode.\n");
+            returnCode = 1;
+        }
+        // Don't subsample when using AVIF_MATRIX_COEFFICIENTS_IDENTITY.
+        input.requestedFormat = AVIF_PIXEL_FORMAT_YUV444;
+        // Quantizers.
+        if (minQuantizer > 0 || maxQuantizer > 0 || minQuantizerAlpha > 0 || maxQuantizerAlpha > 0) {
```

Comment:
Nit: It may be better to use `AVIF_QUANTIZER_LOSSLESS` instead of 0 here to match line 824. I didn't make this change because it may require running clang-format on this file.

Note: The error message in line 821 should use 0 though because that's what's used in command lines.